interactive/api/static/examples/CHAPMAN/camp_data/species.json

```
@@ -3,8 +3,7 @@
     {
       "name": "M",
       "type": "CHEM_SPEC",
-      "tracer type": "CONSTANT",
-      "description": "Third-body molecule. This is any molecule present in the system."
+      "tracer type": "CONSTANT"
```

```suggestion
      "tracer type": "THIRD_BODY"
```

It really should be this. If this isn't working then it maybe the wrong version of musica is still being used on your machine

---

THIRD_BODY now works with musica 0.7.3, and CHAPMAN runs to completion.